packages/ap-storage/alliance_platform/storage/migrations/0004_asynctempfile_moved_to_location.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.4 on 2023-08-14 04:21
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("alliance_platform_storage", "0003_alter_asynctempfile_original_filename"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="asynctempfile",
+            name="moved_to_location",
+            field=models.TextField(blank=True),
+        ),
+    ]
```

Comment:
May as well squash migrations for the code move

Reply:
I wasn't sure about this - need to work out how to migrate an existing site, maybe easier if migrations still exist but can just update migration table to change the app name